### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by adiroiban**
This is to have a "Standard" security document.

Just link it to twisted/twisted.

---------

I have also enabled security scannind and deps security for this repo, from GitHub.

### CODE REVIEW

**Comment by adiroiban** [comment]
Looks like GitHub Actions YAML needs to be updated.... I will try to do it later

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Security Policy
+
+twisted/incremental project uses the same security policy as [twisted/twisted](https://github.com/twisted/twisted).
+
+For more details please check the [Twisted securty process](https://github.com/twisted/twisted?tab=security-ov-file#readme)
```
Comment (hynek) [inline SECURITY.md]:
```suggestion
The twisted/towncrier project uses the same security policy as [twisted/twisted](https://github.com/twisted/twisted).

For more details, please check the [Twisted security process](https://github.com/twisted/twisted?tab=security-ov-file#readme).
```

**Comment by adiroiban** [comment]
Thanks for the review. Sorry for the copy paste :) I will merge.

**Comment by adiroiban** [comment]
We now have this in the GitHUb UI

image

image

**Pull request merged**